src/mpyl/project.py

```diff
@@ -392,6 +392,26 @@ def from_config(values: dict):
         return Docker(host_name=values["hostName"])
 
 
+@dataclass(frozen=True)
+class BuildArgs:
+    plain: list[KeyValueProperty]
```

Comment:
I'm wondering why this is nested under args. Is there any other flavor than plain?